client/signup/steps/domains/index.jsx

```diff
@@ -287,9 +288,14 @@ class DomainsStep extends Component {
 		 * We want to pre load an experiment to show a plan upsell modal in the plans step
 		 */
 		if ( ! isPurchasingItem ) {
-			loadExperimentAssignment( 'calypso_gf_signup_onboarding_free_free_dont_miss_out_modal_v3' );
+			loadExperimentAssignment( 'calypso_gf_signup_onboarding_free_free_dont_miss_out_modal_v3', {
+				isEligible: flowName === 'onboarding',
```

Comment:
Are we allowed to use `isEligible` [with](https://github.com/Automattic/wp-calypso/blob/trunk/packages/explat-client/README.md#api-explatclientloadexperimentassignment) `loadExperimentAssignment`? I see it is [available](https://github.com/Automattic/wp-calypso/blob/trunk/packages/explat-client-react-helpers/README.md#api-useexperimentexperiment_name) for `useExperiment`, but I wasn't sure if this works with this.

Reply:
Good catch @aaronyan I have fixed this by 0f93751